include/taco/tensor.h

```diff
@@ -916,12 +924,32 @@ struct TensorBase::Content {
   Grid partition;
   std::vector<TensorDistribution> distribution;
 
+  // DISTAL runtime stuff.
+  DISTALRuntime::LegionTensor legionTensor;
+  std::shared_ptr<ir::Module> legionModule;
+
   Content(std::string name, Datatype dataType, const std::vector<int>& dimensions,
           Format format)
       : dataType(dataType), dimensions(dimensions),
         storage(TensorStorage(dataType, dimensions, format)),
         tensorVar(TensorVar(util::getUniqueId(), name, Type(dataType,convert(dimensions)),format)) {
           uniqueId = tensorVar.getId();
+
+          if (dataType.isFloat()) {
+            this->legionModule = std::make_shared<ir::Module>();
+            auto runtime = Legion::Runtime::get_runtime(Legion::Processor::get_executing_processor());
+            auto ctx = Legion::Runtime::get_context();
+            // TODO (rohany): Add a switch-case templated on Legion max dim here.
+            if (dimensions.size() == 1) {
+              // TODO (rohany): Fields of this tensor need to be passed through...
+              //  For now, we know that FID_VAL = 0.
+              legionTensor = createDenseTensor<1, double>(ctx, runtime, dimensions, 0);
```

Comment:
I don't know the cleanest way to plumb information down about what field to use for the TACO tensors. This is another reason that it might be better to use `TensorVar`s, and "bind" `LegionTensor` objects to a particular tensor var when invoking a method.